vendor/assets/javascripts/remote_upload.js

```diff
@@ -0,0 +1,22 @@
+Cmsify.remoteUpload = function(form, elementToClone, elementToInsertBefore, callback) {
+  form.dropzone({
+    dictDefaultMessage: 'Drop files or click here to upload',
+    init: function() {
+      this.on('success', function(req, res) {
+        var clone = $($.clone(elementToClone[0] || elementToClone)); //jquery apparently wraps tr elements in an array
```

Comment:
Please prefix any variables that represent jQuery elements with the `$` character. So `$clone`, `$input`, etc.